rosidl_generator_py/test/test_byte_array.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 Open Source Robotics Foundation, Inc.
```

Comment:
```suggestion
# Copyright 2022 Sony Group Corporation.
```

Reply:
Updated in https://github.com/ros2/rosidl_python/pull/184/commits/2a61cfccec49001845bffed5700acee530c202ab